assets/js/components/ExecutionResults/ExecutionResults.test.jsx

```diff
@@ -313,6 +313,47 @@ describe('ExecutionResults', () => {
     expect(screen.queryByText(checkID2)).toBeNull();
   });
 
+  it("should render ExecutionResults with saved 'passing' filter", async () => {
+    const {
+      clusterID,
+      clusterHosts,
+      checks: [checkID1, checkID2],
+      loading,
+      catalog,
+      error,
+      executionLoading,
+      executionData,
+      executionError,
+      executionStarted,
+    } = prepareStateData('passing');
+
+    renderWithRouter(
+      <ExecutionResults
+        clusterID={clusterID}
+        clusterName="test-cluster"
+        clusterScenario="hana_scale_up"
+        cloudProvider="azure"
+        clusterHosts={clusterHosts}
+        catalogLoading={loading}
+        catalog={catalog}
+        executionStarted={executionStarted}
+        catalogError={error}
+        executionLoading={executionLoading}
+        executionData={executionData}
+        executionError={executionError}
+        savedFilters={['passing']}
+      />
+    );
+
+    expect(screen.getAllByText('test-cluster')).toHaveLength(2);
```

Comment:
What about squashing this test and the previous one in a `it.each` version with two scenarios? 
One scenario that does not pass a `savedFilters` (or an empty one) but it provides the route with querystring, another scenario that does provide the `savedFilters` without querystring.

At the end the expectations are the same for the two tests :thinking:

Reply:
If it calls for too much refactor that you don't feel like addressing, keep it as it is.